crates/solvers/src/domain/liquidity/mod.rs

```diff
@@ -0,0 +1,27 @@
+//! Modelling on-chain liquidity.
+
+pub mod constantproduct;
```

Comment:
Since the enum variant is spelled `ConstantProduct` the file should probably be called `constant_product.rs`.

Reply:
Personally, I dislike the aesthetic of snake_case for module names... Super subjective though, and I think you are right in that I should do this for consistency.

If it is OK with you, I would leave to do this at the top of the PR stack, otherwise I will have some annoying rebase/stash conflicts to deal with.

Reply:
Yeah, totally fine to put a pin it to not having to rebase all your other work. 👌

Reply:
64b7f0f75c74d2c09ea7a59d9bdf4594df82a66e